huaweicloud/services/aom/resource_huaweicloud_aom_application.go

```diff
@@ -187,15 +192,16 @@ func ResourceAomApplicationUpdate(ctx context.Context, d *schema.ResourceData, m
 		return diag.Errorf("error update Application fields %s: %s", opts, err)
 	}
 
-	if response.StatusCode == 200 {
-		return nil
-	}
-
 	defer response.Body.Close()
 	body, err := ioutil.ReadAll(response.Body)
 	if err != nil {
 		return diag.Errorf("error update Application %s: %s", string(body), err)
 	}
+
+	if response.StatusCode == 200 && !strings.Contains(string(body), "err") {
```

Comment:
should use exact matching to avoid the case when `name`  or other fields include “err”